cardano-api/src/Cardano/Api/Orphans.hs

```diff
@@ -691,34 +693,34 @@ instance Crypto.Crypto crypto => ToJSON (VMap VB VP (Shelley.Credential 'Shelley
 
 -----
 
-instance ToJSON (Consensus.StakeSnapshots crypto) where
+instance Crypto.Crypto crypto => ToJSON (Consensus.StakeSnapshots crypto) where
```

Comment:
This instance shouldn't be necessary anymore

Reply:
Removed.